### ISSUE TITLE
OCI Driver ignores measure values

### ISSUE THREAD

**Issue by pvoudouris** [OSGeo/gdal#2687]
I am trying to convert an Oracle spatial table with measured geometries (SDO_GTYPE=3302) to a shapefile running this:
`ogr2ogr -f "ESRI Shapefile" "tshapeARCM.shp" -lco SHPT=ARCM -dim XYM -nlt MULTILINESTRINGM OCI:{username}/{password}@{db_alias}:{table_name} -progress -skipfailures`

The shapefile gets created as PolylineM BUT all M values are 0. Using -dim XYZM will put the M values in the Z dimension

Also ogrinfo reports the geometry as:
`1: {table_name} (3D Unknown (any))
`

**Comment by rouault** [comment]
Yes, the Oracle driver doesn't currently support the M dimension of geometries.

**Comment by pvoudouris** [comment]
Thanks @rouault Do you know if there any plans to support it? Also may I suggest to update the OCI Driver documentation to that effect?

**Comment by rouault** [comment]
> Do you know if there any plans to support it?

Not that I'm aware of. It mostly depends on someone doing it itself or funding someone else to do it.

**Comment by mloskot** [comment]
> may I suggest to update the OCI Driver documentation to that effect?

Looks like a perfect opportunity to contribute with a simple pull request, isn't it.
